aten/src/ATen/native/sparse/SparseTensor.cpp

```diff
@@ -70,7 +70,7 @@ Tensor values_sparse(const Tensor& self) {
 /*** Helper methods ***/
 
 SparseTensor new_sparse(const TensorOptions& options) {
-  AT_ASSERT(!options.is_variable());
+  AT_ASSERT(!options.is_variable());  // TODO: remove this when Variable and Tensor are merged
```

Comment:
same here.

Reply:
I believe there is actually no clear need to check this since only `options.layout()` / `options.device().is_cuda()` / `options.dtype()` are used in this function.